recipes/ckanweb-deploy-exts.rb

```diff
@@ -466,7 +466,7 @@
 end
 
 # Enable DataStore extension if desired
-if "yes".eql? node['datashades']['ckan_web']['dsenable'] then
+if ["yes", "True", "true"].include? node['datashades']['ckan_web']['dsenable'] then
```

Comment:
please also add
```"t", "T", "y", "Y",```

Reply:
Simplified by forcing the value to lowercase.